### ISSUE TITLE
Selenium webdriver about net error with geckodriver 0.16.1

### ISSUE THREAD

**Issue by nine-inch-rails** [SeleniumHQ/selenium#4003]
## Meta -
OS:  
Ubuntu 14.04, Linux 3.13.0 64 bit
Selenium Version:  
3.4.0
Browser:  
Firefox 52.0.1

I'm having an issue with running rspec tests with a combo of Selenium 3.4.0, geckodriver 0.16.1, and firefox 52.0.1 on ubuntu 14.04 LTS and linux 3.13.0 64 bit

Browser Version:  
52.0.1 64 bit

## Expected Behavior -
Automated test connects to server
## Actual Behavior -

Selenium::WebDriver::Error::UnknownError:
Reached error page: about:neterror?e=dnsNotFound

**Comment by lmtierney** [comment]
Please provide [concise reproducible test case](http://sscce.org/) so that we can act on this issue. Once this is done the issue will be re-opened. See [CONTRIBUTING.md](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#issue-contributions)

**Comment by swb0417** [comment]
Here is a "me too" response. Windows 10, Python 2.7, Selenium 3.4.2, Geckodriver 0.16, Firefox 53. Code to reproduce the issue:

```
import sys
from selenium import webdriver
import time
import re
print 'ITEM: ' + sys.argv[1]
print 'MAX BID: ' + sys.argv[2]
itemidurl = "http://www.ebay.com.au/itm/" + sys.argv[1] + "?&autorefresh=true"
maxbidebay = sys.argv[2]
driver = webdriver.Firefox()
driver.get('https://signin.ebay.com.au/ws/eBayISAPI.dll?SignIn&ru=http%3A%2F%2Fwww.ebay.com.au%2F')
time.sleep(10) #Don't really need this
elements = driver.find_elements_by_class_name("fld")
elements[2].send_keys("your username")
elements[3].send_keys("your password")
button = driver.find_element_by_id("sgnBt")
button.click()
time.sleep(5)
print itemidurl
try:
    driver.get(itemidurl)
except:
    print "Boom!"
```

Take out the try/except and you will get the error. Use any valid Ebay item number on the command line. Technically you don't have to make the code sit through the login sequence either...

Here is a item number that is valid for a few days yet, if you are into guitar amps :-)
162528324648

**Comment by cgoldberg** [comment]
"me too" responses that don't add anything to the issue are discouraged (IMHO).

> Take out the try/except and you will get the error. 

or you could just paste the error you are getting and save everyone time

> Technically you don't have to make the code sit through the login sequence either

Then why are you including it in the example?  Same goes for spurious sleeps, prints, unused imports, and command line parsing.  It all just adds noise and makes understanding the conditions to reproduce more difficult.

a *"concise reproducible test case"* is what was asked for.

**Comment by swb0417** [comment]
Wow! I have provided a way to reproduce the error, This is on Ebay, I can't really help it that auctions end and that I can't provide fully qualified URLs. The code I provided takes 10 seconds to copy and paste. You must really think highly  yourself that 20 seconds to copy and paste some code and run it is asking too much of your precious time... I sure hope your wedding night with King Donald is all you hoped for Princess  Goldberg
